src/main/resources/config.properties

```diff
@@ -6,3 +6,4 @@ LEADTIME_URL=http://leadtimemicroservice:8060
 CYCLETIME_URL=http://cycletimemicroservice:8070
 VELOCITY_URL=http://velocitymicroservice:8090
 VIP_URL=http://microservice_vip:5000
+BDCONSISTENCY_URL=http://microservice_BDC:5001
```

Comment:
underscore should not be used. Java does not like underscore at all. Please could we remove it?

Reply:
where is this in the docker compose file?